net-misc/rabbitmq-server/files/rabbitmq-server.init-r4

```diff
@@ -0,0 +1,26 @@
+#!/sbin/openrc-run
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+depend() {
+	need net
+	use dns epmd
+}
+
+start() {
+	ebegin "Starting RabbitMQ"
+	/sbin/start-stop-daemon \
```

Comment:
Ebuild creates user and group `rabbitmq`. I would suggest to use them.
```
--user rabbitmq --group rabbitmq
```

Reply:
User switching is done by `files/rabbitmq-script-wrapper`.

Reply:
Oh, I see, sorry for the noise.